src/solvers/LevenbergMarquardt.jl

```diff
@@ -40,6 +40,7 @@ then the keyword `jacobian_tangent_basis` below is ignored
 * `β` – parameter by which the damping term is multiplied when the current new point is rejected
 * `initial_residual_values` – the initial residual vector of the cost function `f`.
 * `initial_jacobian_f` – the initial Jacobian of the cost function `f`.
+* `jacobian_tangent_basis` - [`AbstractBasis`](@ref) specify the basis of the tangent space for `jacobian_f`.
```

Comment:
```suggestion
* `jacobian_tangent_basis` - [`AbstractBasis`](@ref https://juliamanifolds.github.io/ManifoldsBase.jl/stable/bases/#ManifoldsBase.AbstractBasis) specify the basis of the tangent space for `jacobian_f`.
```
I've added direct link because it's documented in ManifoldsBase.jl docs. Otherwise LGTM :+1: .